mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
+        times = evoked.times[peaks]
+        if len(times) == 0:
+            times = evoked.times[gfp.argmax()]
+
+    if times == "auto":
```

Comment:
for backward compat you'll probably need `if times is None or times == 'auto':`; it should probably come before `if times == 'peaks`; and `if times == 'peaks'` should be `elif times == 'peaks':`